test/test-network/systemd-networkd-tests.py

```diff
@@ -22,7 +23,11 @@
 dnsmasq_pid_file='/run/networkd-ci/test-test-dnsmasq.pid'
 dnsmasq_log_file='/run/networkd-ci/test-dnsmasq-log-file'
 
+networkd_bin='/usr/lib/systemd/systemd-networkd'
 wait_online_bin='/usr/lib/systemd/systemd-networkd-wait-online'
+networkctl_bin='/usr/bin/networkctl'
```

Comment:
I'm not sure these paths (including `wait_online_bin`) should be hardcoded. I think we could either use something like `@rootlibexecdir@/systemd-networkd` and replace it with `m4` (using `meson_apply_m4`) or assume that when, for example, `--build-dir` isn't passed explicitly, networkd and all its components have been installed properly and avoid overriding them.

Reply:
Having said that, as far as I know, there is exactly one place where the test is run regularly so maybe it's OK for the time being to keep things simple and hardcode these paths as long as it works on CentOS CI. I'd wait for @mrc0mmand.

Reply:
Currently, the test is not installed. So, if it is worth to install the test, then I will update the hard-coded paths.